src/features/document/document_feature.py

```diff
@@ -174,3 +175,18 @@ def remove(address: str, user: User = Depends(auth_w_jwt_or_pat)):
     - str: "OK" (200)
     """
     return remove_use_case(user=user, address=address)
+
+
+@router.get("-existence/{address:path}", operation_id="document_check", response_model=bool, responses=responses)
```

Comment:
is it possible to define the endpoint to be /address/exists? Not sure if fast api is able to interpret the url parameter address in this case. But the url /documents/address/exists is a bit easier to read than /documents-existence/address IMO
```suggestion
@router.get("{address:path}/exists", operation_id="document_check", response_model=bool, responses=responses)
```